src/services/video/video.service.ts

```diff
@@ -51,20 +54,24 @@ export class VideoService {
 
             const entity = await videoRepo.findOne({
                 where: { id },
-                relations: ['uploader'],
+                relations: ['uploader', 'author'],
             });
 
             if (!entity) {
                 throw new NotFoundException();
             }
 
+            if (video.author && normalizeString(video.author) !== normalizeString(entity.author.name)) {
```

Comment:
чет сложно, лучше так

```suggestion
            if (normalizeString(video.author) !== normalizeString(entity.author.name)) {
```